jx_sqlite/setop_table.py

```diff
@@ -249,42 +249,54 @@ def _accumulate_nested(rows, row, nested_doc_details, parent_doc_id, parent_id_c
                     previous_doc_id = doc_id
                     doc = None
                     curr_nested_path = nested_doc_details['nested_path'][0]
-                    
-                    for i, c in nested_doc_details['index_to_column'].items():
-                        value = row[i]
-                        if value == None:
-                            continue
-                        if value == '':
-                            continue
+                    index_to_column = nested_doc_details['index_to_column'].items()
+                    if index_to_column:
```

Comment:
This is here for the case when we have nested documents , we don't want output_data to have None when we were a expecting list of objects. It will fill output_data on recursive call to the same in next for loop.